setup.py

```diff
@@ -14,17 +13,38 @@
     'pytest-httpserver',
     'moto',
 ]
+doc_require = [
+    'Sphinx',
+    'sphinx_rtd_theme',
+    'sphinx-click',
+    'sphinxcontrib-plantuml',  # `apt-get install -y plantuml`
+    'recommonmark',
+    # version related dependencies
+    'setuptools',
+    'setuptools_scm[toml]',
+]
 
 extras_require = {
     'performance': ['ciso8601', 'bottleneck'],
     'interactive': ['matplotlib', 'fiona'],
     'distributed': ['distributed', 'dask[distributed]'],
-    'doc': ['Sphinx', 'setuptools'],
+    'doc': doc_require,
     'replicas': ['paramiko', 'sshtunnel', 'tqdm'],
     'celery': ['celery>=4', 'redis'],
     's3': ['boto3'],
     'test': tests_require,
+    'cf': ['compliance-checker>=4.0.0'],
```

Comment:
`cf` doesn't mean anything to me. It might mean something to everyone else, though! Is `cf` the right word for this?

Reply:
This is NetCDF/CF format compliance checker, only used in tests. So, `cf` is probably not so great, maybe `testall`, `testfull`, `testcf`